src/translations/sqlb_es_ES.ts

```diff
@@ -5170,18 +5170,19 @@ Aviso: aún no se han cargado todos los datos desde la tabla debido al mecanismo
     <message>
         <location filename="../PreferencesDialog.ui" line="576"/>
         <source>Max Recent Files</source>
-        <translation type="unfinished"></translation>
+        <translation>Máximo de archivos recientes</translation>
     </message>
     <message>
         <location filename="../PreferencesDialog.ui" line="596"/>
         <source>Prompt to save SQL tabs
 in new project file</source>
-        <translation type="unfinished"></translation>
+        <translation>Prompt para guardar pestañas SQL
```

Comment:
Aquí _prompt_ no es un tema técnico, es una opción para "Preguntar si guardar pestañas SQL...". La traducción debería ir por ahí. Esto lo implementé yo, pero Justin, que es nativo, me dijo que mejor _prompt_; yo había puesto algo en inglés sencillito como "Ask to" o similar :smile:

Reply:
Ok., yo lo había considerado como algo similar al 'prompt' de la linea de comandos y no veía una traducción que fuese mejor que dejarlo la original.

Así está claro, no lo había visto en la ventana de configuración.

Lo cambio a ""Preguntar si guardar pestañas SQL"